libs/farcaster-provider/src/lib/FarcasterMonitor.ts

```diff
@@ -119,11 +119,11 @@ export class FarcasterMonitor {
     if (userListener.lastCastTimestamp !== null) {
       const storedTimestamp = userListener.lastCastTimestamp;
       const newCasts = filteredCasts.filter((cast) => {
-        return cast.timestamp > storedTimestamp;
+        return cast.timestamp > storedTimestamp && (user.withHashTagOnly === false || cast.text.toLowerCase().includes(TWITCAST_HASHTAG));
       });
       await this.tweetCasts(user, newCasts);
     }
-
+    // Will these be lost any time the server restarts?
```

Comment:
They will be, but that should be fine cause they'll be restored when this class is instantiated (see `init()` method).

I've thought about storing this in the DB, but haven't really found it necessary.